mappings/net/minecraft/predicate/PlayerPredicate.mapping

```diff
@@ -1,10 +1,16 @@
 CLASS net/minecraft/class_4553 net/minecraft/predicate/PlayerPredicate
 	FIELD field_20722 ANY Lnet/minecraft/class_4553;
 	FIELD field_20723 experienceLevel Lnet/minecraft/class_2096$class_2100;
-	FIELD field_20724 gamemode Lnet/minecraft/class_1934;
+	FIELD field_20724 gameMode Lnet/minecraft/class_1934;
```

Comment:
Could you also rename `PlayerPredicate.Builder.gamemode` as well while you're at it?